Formula/exiftool.rb

```diff
@@ -3,9 +3,8 @@ class Exiftool < Formula
   homepage "https://exiftool.org"
   # Ensure release is tagged production before submitting.
   # https://exiftool.org/history.html
-  url "https://cpan.metacpan.org/authors/id/E/EX/EXIFTOOL/Image-ExifTool-12.60.tar.gz"
-  mirror "https://exiftool.org/Image-ExifTool-12.60.tar.gz"
-  sha256 "73dbe06d004c31082a56e78d7f246f2bb0002fbb1835447bc32a2b076f3d32ad"
+  url "https://exiftool.org/Image-ExifTool-12.62.tar.gz"
```

Comment:
I dont think we want to ship dev releases though

![image](https://github.com/Homebrew/homebrew-core/assets/1580956/2594abe4-2c2e-41d5-a707-a5b879e3b509)

Reply:
Ohh, 12.62 is the version that they show on their homepage. I thought they might have bumped it to be the default version.